src/xpublish_tiles/pipeline.py

```diff
@@ -873,11 +875,21 @@ def subset_to_bbox(
             )
         )
 
+        has_discontinuity_x = False
```

Comment:
can we split out this change to a new PR please? It's a lot easier to review and presumably doesn't change any existing snapshot (if it did, I'd want to understand why)